src/v2/Components/Authentication/TermsOfServiceCheckbox.tsx

```diff
@@ -16,12 +16,31 @@ export const TermsOfServiceCheckbox = ({
     <StyledCheckbox {...{ checked: value, error, onChange, onBlur, name }}>
       <Serif color={color} size="3t" ml={0.5}>
         {"By checking this box, you consent to our "}
+        <Link
+          href="https://www.artsy.net/terms"
+          target="_blank"
+          color="blue100"
```

Comment:
I don't know if this is relevant here at all but if you like there's the `brand` color which is purple in v2 contexts and blue in v3 contexts.

Reply:
Wait, so which is preferred purple or blue for links?

Reply:
Sounds like the new v3 way is blue

Reply:
Fixed after discussion in standup.